core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala

```diff
@@ -108,16 +147,33 @@ class SubmissionSupervisor(executionServiceCluster: ExecutionServiceCluster,
     new ThresholdOneForOneStrategy(thresholdLimit = 3)(alwaysRestart)(thresholdFunc)
   }
 
+  private def initGlobalJobExecGauges(): Unit = {
+    try {
+      SubmissionStatuses.allStatuses.foreach { status =>
+        ExpandedMetricBuilder.expand(SubmissionStatusMetricKey, status).asGaugeIfAbsent("current") {
+          globalSubmissionStatusCounts.getOrElse(status, 0)
+        }
+      }
+      WorkflowStatuses.allStatuses.foreach { status =>
+        ExpandedMetricBuilder.expand(WorkflowStatusMetricKey, status).asGaugeIfAbsent("current") {
+          globalWorkflowStatusCounts.getOrElse(status, 0)
+        }
+      }
+    } catch {
+      case NonFatal(e) => logger.warn(s"Could not initialize gauge metrics for jobexec", e)
+    }
+  }
+
   private def initGauge(workspaceName: WorkspaceName, submissionId: UUID): Unit = {
     try {
       WorkflowStatuses.allStatuses.foreach { status =>
         workspaceSubmissionMetricBuilder(workspaceName, submissionId).expand(WorkflowStatusMetricKey, status).asGaugeIfAbsent("current") {
-          currentWorkflowStatusCounts.getOrElse(status, 0)
+          activeWorkflowStatusCounts.get(submissionId).getOrElse(status, 0)
         }
       }
       SubmissionStatuses.allStatuses.foreach { status =>
         workspaceMetricBuilder(workspaceName).expand(SubmissionStatusMetricKey, status).asGaugeIfAbsent("current") {
-          currentSubmissionStatusCounts.getOrElse(status, 0)
+          activeSubmissionStatusCounts.get(workspaceName).getOrElse(status, 0)
```

Comment:
these never get updated after the submission finishes. can we unregister the gauge then?

Reply:
i spent some time looking into "one last update and then unregister" but it was an annoying pain. it doesn't matter so much now that we have the globally-aggregated metrics; if we look at the workflow counts for a submission and they suddenly just stop, it's because the last one succeeded.